solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -187,6 +198,7 @@ contract RandomBeacon is Ownable {
         IERC20 _tToken,
         IStaking _staking
     ) {
+        // TODO: RandomBeacon must be the owner of the sortition pool.
```

Comment:
I take that the sortition pool is a 1:1 relation with the 'project' that uses it? ie. `random-beacon`<->`sortition pool`, `ecdsa`<->`different sortion pool`?

Reply:
Yep, see https://github.com/keep-network/sortition-pools/pull/125 to get more context.